### ISSUE TITLE
Has PBStat been renamed to BNStat?

### ISSUE THREAD

**Issue by bcbrock** [Microsoft/CNTK#1576]
I was having some trouble with the "bnstat" command in an example but couldn't find any references to "bnstat" other than the source code. Was this command called "PBStat" in the past?

https://github.com/Microsoft/CNTK/wiki/Top-level-commands#pbstat-command

If so, and if someone will tweak the documentation, may I ask for a clarification? The documentation includes this bullet:

- `minibatchSize`: the mini-batch size while evaluating, same with training minibatchSize

Does this mean that the `minibatchSize` declared here must be the same as that used with the "train" command?

**Comment by cha-zhang** [comment]
In general we have found BNStat not very useful. 

If you have batch normalization layers in your model, it's perfectly fine to skip this step and accuracy should be similar.

**Comment by bcbrock** [comment]
Thank you.
